@uportal/esco-content-menu/src/components/PortletCard.vue

```diff
@@ -218,6 +218,51 @@ export default {
     font-size: 14px;
   }
 
+  &.custom-card {
+    padding: 5px;
+    padding: var(--content-gridcard-padding, 5px);
+    border: none;
+    border: var(--content-gridcard-border, none);
+    background-color: white;
+    background-color: var(--content-gridcard-bg-color, white);
+    border-radius: 5px;
+    border-radius: var(--content-gridcard-border-radius, 5px);
+    box-shadow: none;
+    box-shadow: var(--content-gridcard-shadow, none);
+    transition: box-shadow 0.25s;
+    width: $PortletCardSizeCustomWidth;
+    width: var(--content-gridcard-size-w, $PortletCardSizeCustomWidth);
+    height: $PortletCardSizeCustomHeight;
+    height: var(--content-gridcard-size-h, $PortletCardSizeCustomHeight);
+
+    &:hover {
+      cursor: pointer;
+      box-shadow: none;
+      box-shadow: var(--content-gridcard-shadow-hover, none);
+    }
+
+    & > .portlet-card-icon {
+      > div {
+        height: 75px;
+        height: var(--content-gridcard-icon-size, 75px);
+        width: 75px;
+        width: var(--content-gridcard-icon-size, 75px);
+        margin-top: 0;
+      }
+    }
+
+    > .portlet-card-title {
+      overflow-x: hidden;
+      overflow-y: visible;
+      text-overflow: ellipsis;
+      font-size: var(--content-gridcard-title-fontsize, 16px);
+    }
+
+    > .portlet-card-description {
+      font-size: var(--content-gridcard-description-fontsize, 16px);
+    }
+  }
+
```

Comment:
Somes css part are duplicated from the original css and more initialized with the same values, so why not applying directly css variables on ?

Reply:
I was trying to avoid changing anything that might affect the other 'sizes'. This way they are encapsulated in the 'custom' size.

Reply:
Ok I understand, but:
- I think that some parts could be for a general purpose margin, padding, font-size
- the property content-gridcard-icon-size could have 2 properties, one for the general purpose and one other for the medium, small, smaller part
- why duplicating the original value and not applying a common default scss variable as value ?

In fact when watching at the code only the  
```css
    width: $PortletCardSizeCustomWidth;
    width: var(--content-gridcard-size-w, $PortletCardSizeCustomWidth);
    height: $PortletCardSizeCustomHeight;
    height: var(--content-gridcard-size-h, $PortletCardSizeCustomHeight);
``` 

have to be in a specific place, all other variables can be merged in the original code or adapted to the specific context (small/smaller/medium, etc...)

Reply:
I think this would be a good idea but should be handled in another pull request as an iteration over this new feature.  The point of my addition is really to create a more generic looking card class that pulls in css variables from the portal's css :root definition without affecting the other size styles.

Reply:
@jgribonvald I second handling this in another PR.
With variables in place, I somewhat question the need for the different size properties.
Those could become CSS classes that apply different variables.

Reply:
yes it could, but this was usefull to get on some part (from parent point) breakpoints to apply custom stuff in components, mainly for the auto part !